lib/core/utils/flattened-tree.js

```diff
@@ -123,6 +123,9 @@ axe.utils.getFlattenedTree = function (node, shadowId) {
 axe.utils.getNodeFromTree = function (vNode, node) {
 	var found;
 
+	if (vNode.actualNode === node) {
```

Comment:
makes the additional test pass - specifically when some of our utilities look for the 'html' node in the vrtual tree, it was not being found.